<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QWidget" name="options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="OptionsLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PowerBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>POWER OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SleepBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>SLEEP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProfilesBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>PROFILES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PowerBtn_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
